index.html

```diff
@@ -56,28 +56,24 @@
       <img src="static/img/gp-logo.svg" alt="Greenpeace" class="logo">
     </div>
 
-    <div class="heading">
+    <h1 class="heading">
       Together we can change the world
-    </div>
+    </h1>
 
     <div id="loading">
       <div class="three-quarters" id="spinner-people">loading...</div>
     </div>
 
-    <div id="cta" class="cta">
-      <div id="dropdown" class="dropdown">
-        <a class="btn btn-primary" id="selection" href="https://www.greenpeace.org/international/">
-          <span class="truncate">Go to Greenpeace <span id="country-name">International</span></span> <i id="arrow" class="arrow arrow-down"></i>
-        </a>
-        <div id="dropdown-group" class="dropdown-group">
-        </div>
-      </div>
-      <span class="hidden-tall">or</span>
-      <a class="btn btn-secondary hidden-tall" href="#country-list" id="country">Select another country/region</a>
-    </div>
+    <nav id="cta" class="cta">
```

Comment:
I think this ID is no longer needed, since you removed [that line](https://github.com/greenpeace/planet4-landing-page/pull/65/files#diff-82c17de3ab5c5c0b9ca0bfbc799a3781664c49b4e3e604b795c50020fdd39b33L30)?

Reply:
In fact there are a bunch of IDs that don't seem necessary

Reply:
It's being used [here](https://github.com/greenpeace/planet4-landing-page/pull/65/files#diff-2c6aecf9b86f38a0f87902fa745a3faf16492a851a5d9bc0b5447a7e0f4847b8R31)

Reply:
Ah, my bad 😬